### ISSUE TITLE
NPM packaged does not exists

### ISSUE THREAD

**Issue by AlbertoMonteiro** [ben-girardet/aurelia-summernote#1]
There is no package on NPM https://www.npmjs.com/search?q=aurelia%20summernote&page=1&ranking=optimal

**Comment by ben-girardet** [comment]
That's right. I've planned to publish it in the future but I've never done this myself so I need to learn and do it. Meanwhile you can still install the plugin via npm using the command:

```
npm install ben-girardet/aurelia-summernote
```

or 

```
npm install en-girardet/aurelia-summernote#bs4
``` 

If you want to use the version compatible with bootstrap 4

**Comment by AlbertoMonteiro** [comment]
Thanks for answer, I will try later!!!!

**Comment by AlbertoMonteiro** [comment]
Thanks for the effort @ben-girardet it is working nicely!!!
